packages/cli/generators/project/templates/README.md.ejs

```diff
@@ -1,3 +1,15 @@
 # <%= project.name %>
 
 [![LoopBack](https://github.com/strongloop/loopback-next/raw/master/docs/site/imgs/branding/Powered-by-LoopBack-Badge-(blue)-@2x.png)](http://loopback.io/)
+
+## Installation
+
+Describe the installation steps.
+
+## Basic Use
+
+Describe the usage.
```

Comment:
Can the generator put the default example we are using in our components? Something along the following lines (replace `MyComponent` with the real name):

```ts
this.configure(MyComponent.COMPONENT).to({
 // put the configuration options here
});
this.component(MyComponent);
```